practice-two/.gitignore

```diff
@@ -0,0 +1,4 @@
+.parcel-cache
+dist
+node_modules
+package-lock.json
```

Comment:
please check .editorconfig, does it work? This file is not added empty line at the end.